src/commands/fastlane-deploy.yml

```diff
@@ -0,0 +1,16 @@
+description: |
+  Runs fastlane lanes to deploy to Google Play Store
+parameters:
+  working-directory:
+    description: Working directory to run fastlane commands in
+    type: string
+    default: "."
+  lane-name:
+    description: The name of the lane fastlane will execute
+    type: string
+    default: "deploy"
+steps:
+  - run:
+      name: Run fastlane
+      command: fastlane << parameters.lane-name >>
+      working_directory: << parameters.working-directory >>
```

Comment:
FWIW I typically advocate people avoid adding fastlane as a means to a single end since gradle is itself a good automation tool that android devs and/or build engineers are accustomed too.

Typically I recommend https://github.com/Triple-T/gradle-play-publisher for this type of work. I've had good luck with it over the years. It is typically very fast in addressing issues when google changes things in AGP and the play api's.

Having a "general purpose" android orb like this one implicitly force people to use a new tool like fastlane to accomplish a common task seems ill advised since the orb version update or command won't receive the same level of scrutiny that adding an entirely new build tool/helper would.

Reply:
Hi @trevjonez I decided to use fastlane as it is already supported in the macOS orb and image. The gradle publishing route is something that I would like to dive into as well for a second implementation of deployment. I also included fastlane in the title of anything that uses fastlane so I can later include a gradle implementation without making a breaking change.

Reply:
Thanks for the tips! If there is support behind using a specific deployment method, we can always implement the option in the future.

Since this is a new feature, users can continue using their current deployment methods.